nodes/mainnet.md

```diff
@@ -1,26 +1,26 @@
 <!-- markdownlint-disable MD033 -->
 
-# Mainnet Beta
```

Comment:
please revert all of these instances back to `Mainnet Beta`

Reply:
done in https://github.com/celestiaorg/docs/pull/1525/commits/7be6097074cc56d136ce09a1b3b94a79c9c317c7